src/TaxonomyTerm.php

```diff
@@ -193,31 +193,31 @@ public function providePermissions()
         return array(
             'TAXONOMYTERM_EDIT' => array(
                 'name' => _t(
-                    'TaxonomyTerm.EditPermissionLabel',
+                    __CLASS__ . '.EditPermissionLabel',
                     'Edit a taxonomy term'
                 ),
                 'category' => _t(
-                    'TaxonomyTerm.Category',
+                    __CLASS__ . '.TaxonomyTerm.Category',
```

Comment:
This seems like not enough was removed before adding `__CLASS__` back in

Reply:
Oh true!